ftest.sh

```diff
@@ -217,6 +247,66 @@ dmesg
 df -h
 EOF
 
+# apply patch for https://github.com/avocado-framework/avocado/pull/3076/
```

Comment:
is this patch embedded in the shell script?   Why not just have a separate file?  Is it something we plan to push upstream?

Reply:
It's something that upstream has asked us to test to see if it resolves our issue, so yes, in time this patch should go away.